config/credentials.yml.enc

```diff
@@ -1 +1 @@
-pbuoaqO1R8B5Rxr4YNfKZ1+jhElh+xw+r54fjDd0OxVspFiYA2PuJnfVJ7PYXSBG9QtI9boITLHfUI9hRF/LTeIDJgcXNxCawijEEfJikaa+sVWJWCbUoRfaLWvIcM685dlv7LQjdxcaG27NCXajXCgUdTqzyPjAQBGjgm9XoSOWanXgDEDIK2W+Mas7j8NSf9jewwz0rjYI06Dhyp4ZhH7YVXqIn+tKNaIXlWV7dLtrjsUA7TzWXNJ4IWzF4rvvStO9hNhflsfOTIlfPXWBeHS0wki9yxhemFrx3qHFOP5EScGH7OvuB3+5cwVpERoCUEQf+1+p/0ktYva7WLvynhzCVzm30NIVvYh6klDJh4xQ/6gTFBVhztJ7fe/7nusHIMa4U/ob9OLoOh4C2evBQdpIIwej--+f8BOevvq3ceuKK2--7kWOVUUENeVPznqJlEF8OQ==
\ No newline at end of file
+lm7hU15RPDAELbHXXqGnSzR1nZ2pxZqVszMQCEfC+ng9MHEZ6mIh+Ty9W5/efftemRYn9ZznWg16MlmD9Oi0a+5Kw0+toaIqmjXIlyrnt+CcfQ7XtD8bA5acD+ADSBO8gJozfLNuZoq+ZDk/l/miti3+HEOPZ9sZinBmC2u07NGHM9gbYX/n5Nmb3uYF6ldk1C+lNaAOgF8LPqqcmCP13520w8e2xPoSgeDAsu8ZjcK4U17N5LGFvoHGyTRg+xmKLEsSx97zNRI2Z5b65AWiiMN+yfNGmqYMSwxPOLYFyWjgTb6Qyk1UxTyibtqWRe15ZEZ24lalGOwdkA0y/n0Rcg85OdznDY/127CkdRwRbG89kfefCTIqEp+vv/o3F5cBJFSrhRRoMnRpPSrt7Ui3xnSTHNbT3Km6vq7ePHx+VJnJW2OwTkrhvyE21dbKSgyVXTvt7YV8rlsFcALejjr0nN9IqoxN4HS5JCeCU2qGpZle74kyRnvxDAt0ZGKDaiP4qWhI9gclCHG2EM+dQAHE1XHz5OtTSIfiwg8CMOWQg8TVC9bmwoGtwMN7H+Vb8TBrQSdM0tZG8w+OAxwDVRmkLpl1Vn6wAU9K00cZaH4=--Ah+gOA0zqHVtGfvJ--x0HyFilI/g7mY7ZK/PdxHg==
```

Comment:
Ideal que isso fique no `.gitignore
`
Colocar tambem uma instrucao de como gerar um novo credentials, para usar o `devise_jwt_secret_key` no README, perdi um tempinho aqui pra conseguir

Reply:
Adicionei e coloquei uma descrição no readme de como fazer, confesso que também tinha esquecido mas achei a fonte que eu tinha seguido pra fazer kkkkkk
https://sdrmike.medium.com/rails-7-api-only-app-with-devise-and-jwt-for-authentication-1397211fb97c